README.md

```diff
@@ -11,12 +11,9 @@
 * [0026: Environment Variable Configuration Of Buildpack](./text/0026-environment-variable-configuration-of-buildpacks.md)
 * [0027: Common Logging Levels for Buildpacks](./text/0027-log-levels.md)
 * [0029: Semantic Versioning of Buildpacks and Builders](./text/0029-semantic-versioning.md)
-* [0030: Buildpackless Builders](./text/0030-buildpackless-builders.md)
-* [0031: Liberty Buildpack](./text/0031-liberty-buildpack.md)
```

Comment:
0031: Liberty Buildpack is still in the works. That shouldn't be accepted yet.

Reply:
Gotcha that is my mistake will fix

Reply:
@dmikusa-pivotal should be fixed